src/buildstream/_remotespec.py

```diff
@@ -112,6 +113,9 @@ def __init__(
         # The grpc credentials object
         self._credentials: Optional[ChannelCredentials] = None
 
+        # Various connection parameters for grpc connection
+        self._connection_config: Optional[int] = connection_config
```

Comment:
```suggestion
        self._connection_config: Optional[MappingNode] = connection_config
```

Reply:
You are right, if this is set in project config, then it clearly needs a version bump. I was assuming (perhaps incorrectly) that the most useful way to use this is through user config where you override project CAS with your own config and set timeout there if you're having connection issues.

Reply:
I fixed this wrong, now it's correct.